### ISSUE TITLE
Leave a review - text is not visible

### ISSUE THREAD

**Issue by patrickxkim** [KjellConnelly/react-native-rate#48]
I have a strange behaviour. Everything works fine, but when a user clicks to leave a review, the optional comment review section is all bg color when you type your review in. The title section comes out fine with black text. But the review section text is the same as bg color and you cant see what you are typing. I tested it out in several devices and same issue... please help! I even tried react-native-store-review and same issue there...

**Comment by KjellConnelly** [comment]
I'm guessing this is iOS13 bug behavior (maybe dark mode problems) since this module accesses the native APIs. Don't have time to get into it now, and still using iOS 12 currently. 

https://stackoverflow.com/questions/57763132/skstorekitreviewcontroller-and-darkmode

I found this on SO. Possible solution is to change BG color of your app while waiting for review to open. If it's my module's problem, and not Apple's, let me know.

**Comment by patrickxkim** [comment]
not sure its a ios13 only issue.  I've tried it on a phone not on dark mode, same thing.  

I do believe its a Apple issue with the SKStoreReviewController and not yours, but I just wanted to see if others were experiencing this too.  The bg color of my app has no bearing on the pop up that is used.

**Comment by ElieMyIdea** [comment]
Hi,

I'm having the same issue on IOS 13.1.2 (it was working well on ios 12)

**Comment by nikolas7892** [comment]
can't we just add `	<key>UIUserInterfaceStyle</key>
	<string>Light</string>` to the info.plist of this project?

**Comment by KjellConnelly** [comment]
According to someone from SO: "This issue was resolved in iOS 13.2. The blur effect is dark, just like other system alerts when Dark Mode is enabled."

**Comment by gusgard** [comment]
I'm using the latest version and we are having this problem (with light and dark mode)

https://user-images.githubusercontent.com/2577356/109709471-4db78a00-7b7b-11eb-857f-c51aba61e1fa.mp4


We are also using `<key>UIUserInterfaceStyle</key> <string>Light</string>`, does anyone knows if it is related to this library?

**Comment by KjellConnelly** [comment]
Hi @gusgard ,

Although this is a similar problem, I believe it is different. The original problem was that the initial pop up was off color. In your video, it seems just the body of the text input is incorrectly colored.

This is an Apple issue, as third party developers have no access to modifying anything after the initial pop up, and before all is dismissed. I'd suggest looking for this topic outside of the react native module/community as it may be another bug on Apple's end that others have spotted. 

You can try removing that key. I wouldn't be surprised if it was the problem.

**Comment by gusgard** [comment]
@KjellConnelly thanks for your feedback 👍

**Comment by gusgard** [comment]
Might or might not be related to this github issue but this https://developer.apple.com/forums/thread/127787 help me to fix the issue I was having with the Apple review component

```
if #available(iOS 13.0, *) {
--
overrideUserInterfaceStyle = .light
}
```
